src/pivotal-ui/components/meta/need-help-orgs.hbs

```diff
@@ -2,7 +2,7 @@
 
   <div class="row">
     <div class="col-xs-7 col-xs-offset-1 txt-c">
-      <img alt="support avatar" class="avatar" height="60" src="styleguide/orgs-svgeople-avatar.svg" width="60">
+      {{svg "@npmcorp/pui-css-meta/orgs-svgeople-avatar.svg" alt="support avatar" class="avatar" height="60" width="60"}}
```

Comment:
`fromPackage` means we're using an `svg` helper. That helper allows us to reference directly from within the package.

Using the image tag means we're referencing a static image and don't know where we're getting it.

So, since we're using `fromPackage`, we need to be able to reference a local image to the specific module.